README.md

```diff
@@ -155,6 +149,18 @@ This knife plugins requires >= Chef 11.0.0. More details about Knife plugins can
 
 ## Nodes
 
+### Bootstrapping Windows Nano Server
+
+Windows Nano Server cannot install Chef via an MSI. Instead Nano supports the `appx` package format. Until Omnitruck strarts serving appx packages automatically to nano, you will need to provide an appx package URL to the `--msi-url` argument:
+
+```
+knife bootstrap windows winrm 192.168.137.3 -x vagrant -P vagrant -N nano1 --msi-url https://s3-us-west-2.amazonaws.com/nano-chef-client/chef-12.14.60.appx
+```
+
+### Powershell version
+
+The node must be running Powershell version 2.0 or higher. Windows 2008 R2 and greater come with a compatible version of Powershell installed by default. If you are trying to bootstrap Windows Server 2008 SP2 or lower, you must install the Windows Management Framework 2.0 or higher in order to succesfully use `knife bootstrap windows winrm`.
```

Comment:
`lib/chef/http/http_request.rb:        @headers["X-Chef-Version"] = ::Chef::VERSION` 

There's more than that in the header. Still looking for where that comes from.